### PR TITLE
Create crt-guest-dr-venom-stock.slangp

### PR DESCRIPTION
**Pull request by lfanog**
A stock version of crt-guest-dr-venom shader for systems like N64, GameCube, Playstation, Saturn, Dreamcast etc.

### CODE REVIEW

**Comment by hizzlekizzle** [comment]
Can you move that to the 'presets' directory?

**Comment by lfanog** [comment]
> 
> 
> Can you move that to the 'presets' directory?

How can I do it? Have to move the crt-guest-dr-venom-stock.glslp too.

**Comment by hizzlekizzle** [comment]
Just move it to the new location and change the file references to match. I just try to keep the unofficial variants in the CRT folder to a minimum and put special-case presets in the presets folder instead.

**Comment by hizzlekizzle** [comment]
you'll also need to change all instances of "shaders/guest" to "../crt/shaders/guest". A simple find-and-replace should do it.

File: crt/crt-guest-dr-venom-stock.slangp [hunk 1]
```diff
@@ -0,0 +1,78 @@
+shaders = 12
+
+shader0 = ../stock.glsl
```
Comment (CatGreen90) [inline crt/crt-guest-dr-venom-stock.slangp]: Shouldn't this be stock.slang?

Comment (lfanog) [reply crt/crt-guest-dr-venom-stock.slangp]: I fixed. Thanks!

**Pull request merged**